lib/features/artifacts.js

```diff
@@ -142,7 +153,9 @@ export default class Artifacts {
       var res = yield waitForEvent(putReq, 'response');
```

Comment:
using abort timeout is nice... but uploads shouldn't hang they need to be retried if they timeout (< 1 mb/s for 60s).
Or if network connection is broken...